### ISSUE TITLE
inf and nan not supported

### ISSUE THREAD

**Issue by kohr-h** [uiri/toml#200]
Infinity and Not-a-Number are described in the [Float section](https://github.com/toml-lang/toml#float) of the TOML spec.

`toml` doesn't recognize them as floating point numbers and tries to use `int` for conversion as can be seen in this minimal example:
```python
>>> toml.loads("""
... a = inf
... """)
Traceback (most recent call last):
...
[...]/site-packages/toml.py", line 664, in _load_value
    v = int(v)
ValueError: invalid literal for int() with base 10: 'inf'
```
Same error for `nan`, and also for arrays containing them.

I'm using `toml` 0.9.4.

**Comment by kohr-h** [comment]
Fixed on `master`, sorry for the noise.

**Comment by kohr-h** [comment]
> This support should also be present in the latest release - `toml` 0.9.6 :)

Hm, I tried that, but it gave the same error. I think the relevant commit dates to before that release, but got merged after it and is not part of 0.9.6.
